README.md

```diff
@@ -1 +1,144 @@
-# school_library
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <br/>
+
+  <h3><b>school_library</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Usage](#usage)
+- [👥 Author](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowlednpments](#acknowlednpments)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 school_library<a name="about-project"></a>
+
+**school_library** The project is a school library management system that aims to facilitate the organization and access to resources in a school's library. It utilizes object-oriented programming in Ruby to create classes for various entities, such as students and teachers, representing users within the school environment. 
+
+## 🛠 Built With <a name="built-with"></a>
+RUBY
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> Key Features
+- Implement classes and objects in Ruby
+- Demonstrates concepts of inheritance, encapsulation, and abstraction through well-defined classes and their relationships, 
+- Efficient and user-friendly library system for educational institutions
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+
+### Prerequisites
+
+In order to run this project you need:
+
+  ruby > 3.0.1
+  code-runner
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/binodbhusal/school_library.git
+```
+
+### Usage
+
+To run the project, execute the ruby file with any code runner of your choice.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Binod Bhusal**
+
+- GitHub: [@githubhandle](https://github.com/binodbhusal)
+- Twitter: [@twitterhandle](https://twitter.com/Binod_ironLad)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/binodbhusal)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- No future features for now.
+
```

Comment:
- [ ] It would be nice if you add some meaningful features in the Future Feature section to make your readme descriptive and professional 👍